.github/workflows/build.yml

```diff
@@ -106,11 +117,19 @@ jobs:
 
           mkdir build
 
-          # TODO signing prototype -- starting very small
           shouldSign="$(jq <<<"$json" -L.scripts 'include "doi"; build_should_sign')"
           [ "$shouldSign" = 'true' ] || [ "$shouldSign" = 'false' ] || exit 1
-          echo "shouldSign=$shouldSign" >> "$GITHUB_ENV"
+          echo "shouldSign=$shouldSign" >> "$GITHUB_OUTPUT"
 
+          if [ "$shouldSign" = 'true' ]; then
+            tags=$(jq <<<"$json" -L.scripts 'include "provenance"; [ tags ]')
+            echo 'tags<<EOF' >> $GITHUB_OUTPUT
+            echo "$tags" >> $GITHUB_OUTPUT
+            echo 'EOF' >> $GITHUB_OUTPUT
```

Comment:
https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/workflow-commands-for-github-actions#multiline-strings

Hmm, not the syntax I would have went with

Reply:
Fixed!

Reply:
Ha! This new way is slightly more readable